setup.cfg

```diff
@@ -1,2 +0,0 @@
-[bdist_wheel]
-universal = 1
```

Comment:
No longer needed?

Reply:
I think it is only needed if we are distributing a wheel for Python 2 and 3: https://packaging.python.org/guides/distributing-packages-using-setuptools/#universal-wheels